docs/usage.md

```diff
@@ -183,9 +183,9 @@ The above pipeline run specified with a params file in yaml format:
 nextflow run nf-core/rnafusion -profile docker -params-file params.yaml
 ```
 
-with `params.yaml` containing:
+with:
 
-```yaml
+```yaml title="params.yaml"
```

Comment:
Maybe it's just my browser, but how do you see this title?

Reply:
I don't see it, but I think that is intentional, but indeed a bit weird